compute/admin_guide/vulnerability_management/scan_reports.adoc

```diff
@@ -144,17 +145,17 @@ Those which were found are displayed under the *Process info* tab.
 image::scan_reports_process_info.png[width=600]
 
 
-=== Per-finding timestamps
+=== Per-finding Timestamps
 
-Prisma Cloud's image scan reports show the following per-vulnerability timestamps:
+The image scan reports of Prisma Cloud show the following per-vulnerability timestamps:
 
 * Age of the vulnerability based on the discovery date.
 This is the first date that the Prisma Cloud scanner found the vulnerability.
 
 * Age of the vulnerability based on its published date.
 This represents the date the vulnerability was announced to the world.
 
-Host scan reports and registry scan reports show the published date only.
```

Comment:
[Discovered date was added to host scan reports](https://docs.paloaltonetworks.com/prisma/prisma-cloud/30/prisma-cloud-compute-edition-release-notes/release-information/release-notes-30-01-update1#:~:text=Discovery%20Date%20for%20Host%20Vulnerability%20Scan%20Results) in Maxwell Update 1.